crypter/vault.go

```diff
@@ -17,6 +18,7 @@ import (
 	"strings"
 
 	"github.com/starkandwayne/goutils/log"
+	"golang.org/x/net/lex/httplex"
```

Comment:
What is httplex?

Reply:
httplex is the library used to determine if http header names/vaules are valid. Im using it to determine if the decryption of the vault creds occurred properly and avoid sending vault invalid headers (it was causing errors where the daemon had to be reset as the vault token header would be invalid and all calls to vault thereafter would fail)

Reply:
Is this because an invalid master password leads to garbage binary data?  If so, I'd recommend a two-pronged approach:

1. Store a "sigil" value in the plaintext before decryption -- something known to SHIELD that is not used for anything other than "did we get a good decrypt".  Bad master passwords will not decrypt properly, so this known value will not match what's in the software.
2. Use a regex to validate the master password to ensure that it only contains printable chars (ASCII space (0x20) to ASCII '~' (0x7e), something like `/^[\x20-\x7e]+$/`